tests/integration/test_foo/test.py

```diff
@@ -0,0 +1,5 @@
+import pytest
```

Comment:
Just to see error output, this test will be removed before merge

Reply:
Output works. But seems there is an excessive \n
```
2021-07-22 13:10:20,962 runner output: import file mismatch:

2021-07-22 13:10:20,962 runner output: imported module 'test' has this __file__ attribute:

2021-07-22 13:10:20,962 runner output:   /ClickHouse/tests/integration/test_foo/test.py

2021-07-22 13:10:20,962 runner output: which is not the same as the test file we want to collect:

2021-07-22 13:10:20,962 runner output:   /ClickHouse/tests/integration/test_merge_tree_s3_restore/test.py

2021-07-22 13:10:20,962 runner output: HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
```